model/installation/registry.yaml

```diff
@@ -0,0 +1,43 @@
+groups:
+  - id: registry.installation
+    type: attribute_group
+    display_name: Installation Attributes
+    brief: >
+      Describes installation attributes.
+    stability: development
+    attributes:
+      - id: installation.id
+        type: string
+        stability: development
+        brief: >
+          A unique identifier representing the installation of an application on a specific device
+        note: |
+          Its value MUST remain the same between launches of the same installation of an application.
```

Comment:
> Its value MUST remain the same between launches of the same installation of an application.

Is installation id generation is something that instrumentation is responsible for? If not, we should avoid normative language or use SHOULD

Reply:
In my opinion the implementation could be provided by the platform or instrumentation. If the platform can provide something fulfilling these criteria/recommendations, the instrumentation libraries might rely on those. E.g. on Android and iOS there are a few options to choose from, however, for desktop JVM/native apps the situation is far less obvious. Ultimately, I would argue that the application developer should have the control to choose one or another.

Since it is partially provided by the platform, I have relaxed it to SHOULD, as you requested.